crates/ark/src/data_explorer/utils.rs

```diff
@@ -67,16 +67,16 @@ pub fn display_type(x: SEXP) -> ColumnDisplayType {
         }
 
         if r_inherits(x, "integer") {
-            return ColumnDisplayType::Number;
+            return ColumnDisplayType::Integer;
         }
         if r_inherits(x, "double") {
-            return ColumnDisplayType::Number;
+            return ColumnDisplayType::Floating;
         }
         if r_inherits(x, "complex") {
-            return ColumnDisplayType::Number;
+            return ColumnDisplayType::Floating;
```

Comment:
This seems pretty weird, but I honestly don't foresee lots of use of complex numbers in the data explorer. But perhaps they should stay as `ColumnDisplayType::Number`.